src/bcf/report/oncoprint.rs

```diff
@@ -361,14 +364,22 @@ pub fn oncoprint(
                 let rec = gene_data.entry(k.to_owned()).or_insert_with(&Vec::new);
                 let filter_canonical = record_tuples
                     .iter()
-                    .filter(|(_, canonical)| *canonical)
+                    .filter(|(_, canonical, _)| *canonical)
                     .collect_vec();
                 match filter_canonical.len() {
                     0 => {
-                        rec.extend(record_tuples.iter().map(|(r, _)| r.clone()));
+                        rec.extend(record_tuples.iter().map(|(r, _, _)| r.clone()));
+                    }
+                    1 => rec.extend(filter_canonical.iter().map(|(r, _, _)| r.clone())),
+                    _ => {
+                        rec.extend(filter_canonical.iter().map(|(r, _, _)| r.clone()));
+                        let transcripts = filter_canonical
+                            .iter()
+                            .map(|(r, _, _)| r.alteration.clone())
+                            .collect_vec();
+                        let positions = filter_canonical.iter().map(|(_, _, p)| p).collect_vec();
+                        warn!("Found more than one variant of gene {} annotated as canonical! The transcripts marked as canonical are {:?} located at {:?}", &k, &transcripts, &positions);
```

Comment:
```suggestion
                        warn!("Found more than one transcript in gene {} annotated as canonical! The corresponding alterations are {:?}, located at {:?}", &k, &transcripts, &positions);
```

Reply:
Great thanks, i oversaw the other suggestion below hence no committing of the suggestions. Should be good to go now though!